assets/app/features/upgrade/i18n/en.json

```diff
@@ -31,6 +31,24 @@
                         "storage": "Storage Health",
                         "free_node_available": "Free node available"
                     }
+                },
+                "upgrade": {
+                    "title": "Begin Upgrade",
+                    "description": "Click on \"Begin Upgrade\" below to start the upgrade process.",
+                    "button": "Begin Upgrade"
+                },
+                "mode": {
+                    "title": "Upgrade Mode",
+                    "description": "After all checks are successful the upgrade mode will be shown here...",
+                    "normal_mode": {
+                        "title": "Your cloud is now prepared to begin the upgrade process in normal mode",
+                        "button": "Proceed in normal mode",
+                        "description": "Long description here"
```

Comment:
Nowhere. I added it when I believed the scope for the change was bigger. Let me remove it....